lib/main_containers.pm

```diff
@@ -55,9 +59,11 @@ sub load_image_tests_docker {
 
 sub load_host_tests_podman {
     if (is_leap('15.1+') || is_tumbleweed || is_sle("15-sp1+")) {
+        my $engine_args = OpenQA::Test::RunArgs->new();
+        $engine_args->{podman} = 1;
```

Comment:
Looks like a good solution, but why you give it a value `1`? 
Could we do something like 
`engine_args->{runtime} = 'docker'` ?